### PR TITLE
DRV 356 recognize runtime env

### PR DESCRIPTION
**Pull request by fireridlle**
### Notes
[DRV-356](https://faunadb.atlassian.net/browse/DRV-356)

New headers applied for each request.

## For NodeJS env
**X-Fauna-Driver-Version**
faunadb-js version (taken from package.json)
**X-Runtime-Environment**
name of runtime env (ex: Netlify, Vercel, etc..) or Unknown for an environment that can't be recognizable
**X-Runtime-Environment-OS**
Name of OS (ex: darwin, linux)
**X-NodeJS-Version**
Version of NodeJS

## For browser
Currently disabled due to CORS error

**X-Fauna-Driver-Version**
faunadb-js version (taken from package.json)
**X-Runtime-Environment**
Browser name
**X-Runtime-Environment-Version**
Browser version
**X-Runtime-Environment-OS**
Name of OS (ex: darwin, linux)

### CODE REVIEW

File: src/_http.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use strict'
 
-var APIVersion = '4'
-
+var pjson = require('../package.json')
+var parse = require('url-parse')
```
Comment (parkhomenko) [inline src/_http.js:4]: I can be mistaken, but I think you have removed this one in the previous PR

File: src/_http.js [hunk 2]
```diff
@@ -84,11 +84,12 @@ HttpClient.prototype.execute = function(method, path, body, query, options) {
 
   var headers = this._headers
   headers['Authorization'] = secret && secretHeader(secret)
-  headers['X-FaunaDB-API-Version'] = APIVersion
-  headers['X-Fauna-Driver'] = 'Javascript'
   headers['X-Last-Seen-Txn'] = this._lastSeen
   headers['X-Query-Timeout'] = queryTimeout
 
+  console.debug('request headers for test ')
```
Comment (parkhomenko) [inline src/_http.js:90]: Was it added just for tests, or you want to keep them intentionally?

File: src/_http.js (hunk 1 above)
Comment (fireridlle) [reply src/_http.js]: done

File: src/_http.js (hunk 2 above)
Comment (fireridlle) [reply src/_http.js]: done

**Review by n400 — COMMENTED** [review]
Let’s wait for Eduardo before merging this

**Review by parkhomenko — APPROVED** [review]

**Comment by evbruno** [comment]
Is there a way to unit test this? (assuming the `util.isNodeEnv()` branch is going to run on CI)

**Comment by fireridlle** [comment]
> Is there a way to unit test this? (assuming the `util.isNodeEnv()` branch is going to run on CI)

yes, it could be. but is it necessary as we will simply set the same env var as we check. for me it would look like reverse checking)

**Review by evbruno — APPROVED** [review]

**Comment by fireridlle** [comment]
just merged the latest master and solve conflict. re-approve please

**Review by parkhomenko — APPROVED** [review]

**Review by tigger9flow — APPROVED** [review]

**Pull request merged**